src/js/trove/make-standalone.js

```diff
@@ -1,8 +1,30 @@
 ({
   requires: [],
-  nativeRequires: ["path", "fs", "requirejs"],
+  nativeRequires: ["path", "fs"],
   provides: {},
   theModule: function(runtime, namespace, uri, path, fs, requirejs) {
+    var NODE_BUNDLED_DEPS_FILE = "build/bundled-node-deps.js";
```

Comment:
We could make some of these part of the config rather than hardcoding them here in the future